edi/models/edi_exchange_consumer_mixin.py

```diff
@@ -45,3 +45,27 @@ def action_view_edi_records(self):
 
     def _edi_generate_records(self):
         raise NotImplementedError()
+
+    @api.model
+    def get_edi_access(self, doc_ids, operation, model_name=False):
+        """We are following the same logic applied on mail.thread and mail.message
+
+        In order to decide if we can create a record is defined by _edi_post_access or
+        by default write, otherwise, the same permission is returned
+        """
```

Comment:
```suggestion
        """Retrieve access policy.

        The behavior is similar to `mail.thread` and `mail.message`
        and it relies on the access rules defines on the related record.

        The behavior can be customized on the related model
        by defining `_edi_exchange_record_access`.
        
        By default `write`, otherwise the custom permission is returned.
        """
```
Is this meaning correct?

_edi_post_access should be renamed to `_edi_exchange_record_access` or similar IMO,